### ISSUE TITLE
Question about training LIBERO

### ISSUE THREAD

**Issue by HeegerGao** [OpenDriveLab/UniVLA#1]
Thanks for open-sourcing this great work! I saw that your results are pretty good on LIBERO benchmarks. May I ask what the approximate L1 loss of the training set is at the end of training for LIBERO? Thanks!

**Comment by retsuh-bqw** [comment]
Thanks for your interest in our work. For Spatial/Object/Goal tasks, the training loss typically converges to below 0.04. On LIBERO-10, it stabilizes around 0.05.

**Comment by HeegerGao** [comment]
Thanks very much for your answer!
